[FP]: aspectjtools 1.9.8

### Package URl

pkg:maven/org.aspectj/aspectjtools@1.9.8

### CPE

cpe:2.3:a:vmware:tools:1.9.8:*:*:*:*:*:*:*)

### CVE

CVE-2014-4199, CVE-2014-4200, CVE-2015-5191, CVE-2016-5328, CVE-2016-7079, CVE-2016-7080, CVE-2018-6969

### ODC Integration

_No response_

### ODC Version

7.0.1

### Description

_No response_

This also happens with aspectjtools-1.9.9.1.jar:
```
[ERROR] aspectjtools-1.9.9.1.jar: CVE-2014-4200(4.7), CVE-2018-6969(7.0), CVE-2014-4199(6.3), CVE-2016-7079(7.8), CVE-2016-5328(5.5), CVE-2016-7080(7.8), CVE-2015-5191(6.7)
```

--- a/core/src/main/resources/dependencycheck-base-hint.xml
+++ b/core/src/main/resources/dependencycheck-base-hint.xml
@@ -421,6 +421,7 @@
         </given>
         <add>
             <evidence type="product" source="hint analyzer" name="product" value="legion-of-the-bouncy-castle-fips-java" confidence="HIGHEST"/>
+            <evidence type="product" source="hint analyzer" name="product" value="legion-of-the-bouncy-castle-fips-java-api" confidence="HIGHEST"/>
         </add>
     </hint>
     <hint>

--- a/core/src/main/resources/dependencycheck-base-suppression.xml
+++ b/core/src/main/resources/dependencycheck-base-suppression.xml
@@ -5003,4 +5003,11 @@
         <packageUrl regex="true">^pkg:maven/javax\.ws\.rs/javax\.ws\.rs-api@.*$</packageUrl>
         <cpe>cpe:/a:eclipse:eclipse_ide</cpe>
     </suppress>
+    <suppress base="true">
+        <notes><![CDATA[
+   FP per issue #4316
+   ]]></notes>
+        <packageUrl regex="true">^pkg:maven/ch\.qos\.reload4j/reload4j@.*$</packageUrl>
+        <cpe>cpe:/a:apache:log4j</cpe>
+    </suppress>
 </suppressions>

--- a/core/src/main/resources/dependencycheck-base-suppression.xml
+++ b/core/src/main/resources/dependencycheck-base-suppression.xml
@@ -4,7 +4,7 @@
         <notes><![CDATA[
             FP per issue #4180
             ]]></notes>
-        <packageUrl regex="true">^pkg:maven/com\.vaadin/vaadin-spring@.*$</packageUrl>
+        <packageUrl regex="true">^pkg:maven/com\.vaadin/vaadin-spring.*@.*$</packageUrl>
         <cpe>cpe:/a:vaadin:vaadin</cpe>
     </suppress>
     <suppress base="true">

--- a/core/src/main/resources/dependencycheck-base-suppression.xml
+++ b/core/src/main/resources/dependencycheck-base-suppression.xml
@@ -4989,4 +4989,11 @@
         <packageUrl regex="true">^pkg:maven/com\.carrotsearch\.thirdparty/simple-xml-safe@.*$</packageUrl>
         <cpe>cpe:/a:simplexml_project:simplexml</cpe>
     </suppress>
+    <suppress base="true">
+        <notes><![CDATA[
+        FP per issue #4208 suppress CVE for explicit version as CPE 'update' field is not supported and CVE is only for the 5.1 beta-1
+   ]]></notes>
+        <packageUrl regex="true">^pkg:maven/com\.hazelcast/hazelcast@(?!5\.1-BETA-1).*$</packageUrl>
+        <cve>CVE-2022-0265</cve>
+    </suppress>
 </suppressions>

--- a/core/src/main/resources/dependencycheck-base-suppression.xml
+++ b/core/src/main/resources/dependencycheck-base-suppression.xml
@@ -669,6 +669,7 @@
             55. cron_project:cron is a linux C application #3548
             56. html2pdf_project:html2pdf is a PHP framework #4021, #4251
             57. shadow_project:shadow is a suite of C-applications on Linux #4237
+            58. Common Desktop Environment is an X11 GUI environment coded in C for Unix #4346 , #4347, #4348
         ]]></notes>
         <filePath regex="true">.*(\.(dll|jar|ear|war|pom|nupkg|nuspec|aar)|pom\.xml|package.json|packages.config)$</filePath>
         <cpe>cpe:/a:sandbox:sandbox</cpe>
@@ -728,6 +729,7 @@
         <cpe>cpe:/a:cron_project:cron</cpe>
         <cpe>cpe:/a:html2pdf_project:html2pdf</cpe>
         <cpe>cpe:/a:shadow_project:shadow</cpe>
+        <cpe>cpe:/a:cde:cde</cpe>
     </suppress>
     <suppress base="true">
         <notes><![CDATA[

--- a/core/src/main/resources/dependencycheck-base-suppression.xml
+++ b/core/src/main/resources/dependencycheck-base-suppression.xml
@@ -485,13 +485,6 @@
     </suppress>
     <suppress base="true">
         <notes><![CDATA[
-        FP per #2499 and #2600
-        ]]></notes>
-        <packageUrl regex="true">^pkg:maven/org\.jetbrains\.kotlinx/kotlinx\-coroutines\-.*$</packageUrl>
-        <cpe>cpe:/a:jetbrains:kotlin</cpe>
-    </suppress>
-    <suppress base="true">
-        <notes><![CDATA[
         This suppresses false positives for Microsoft.VisualStudio.QualityTools.UnitTestFramework.dll.
         ]]></notes>
         <filePath regex="true">.*Microsoft\.VisualStudio\.QualityTools\.UnitTestFramework*\.dll</filePath>
@@ -1569,6 +1562,13 @@
     </suppress>
     <suppress base="true">
         <notes><![CDATA[
+        FP per #2499 and #2600
+        ]]></notes>
+        <packageUrl regex="true">^pkg:maven/org\.jetbrains\.kotlinx/kotlinx\-coroutines\-.*$</packageUrl>
+        <cpe>cpe:/a:jetbrains:kotlin</cpe>
+    </suppress>
+    <suppress base="true">
+        <notes><![CDATA[
         kotlinx-html-jvm is a kotlin DSL for generating HTML and should not be identified as core kotlin (#2346)
         ]]></notes>
         <packageUrl regex="true">^pkg:maven/org\.jetbrains\.kotlinx/kotlinx\-html\-jvm@.*$</packageUrl>
@@ -1579,6 +1579,13 @@
    FP per issue #4212
    ]]></notes>
         <packageUrl regex="true">^pkg:maven/org\.jetbrains\.kotlinx/kotlinx-serialization-core-jvm@.*$</packageUrl>
+        <cpe>cpe:/a:jetbrains:kotlin</cpe>
+    </suppress>
+    <suppress base="true">
+        <notes><![CDATA[
+        false positive per #3274 on org.jetbrains.kotlinx:atomicfu
+        ]]></notes>
+        <packageUrl regex="true">^pkg:maven/org\.jetbrains\.kotlinx/atomicfu@.*$</packageUrl>
         <cpe>cpe:/a:jetbrains:kotlin</cpe>
     </suppress>
     <suppress base="true">
@@ -4498,13 +4505,6 @@
         ]]></notes>
         <packageUrl regex="true">^pkg:maven/org\.mybatis\.generator/mybatis\-generator\-core@.*$</packageUrl>
         <cpe>cpe:/a:mybatis:mybatis</cpe>
-    </suppress>
-    <suppress base="true">
-        <notes><![CDATA[
-        false positive per #3274 on org.jetbrains.kotlinx:atomicfu
-        ]]></notes>
-        <packageUrl regex="true">^pkg:maven/org\.jetbrains\.kotlinx/atomicfu@.*$</packageUrl>
-        <cpe>cpe:/a:jetbrains:kotlin</cpe>
     </suppress>
     <suppress base="true">
         <notes><![CDATA[

--- a/core/src/main/resources/dependencycheck-base-suppression.xml
+++ b/core/src/main/resources/dependencycheck-base-suppression.xml
@@ -4980,4 +4980,11 @@
         <packageUrl regex="true">^pkg:maven/org\.bouncycastle/bc\-fips@.*$</packageUrl>
         <cpe>cpe:/a:bouncycastle:legion-of-the-bouncy-castle:</cpe>
     </suppress>
+    <suppress base="true">
+        <notes><![CDATA[
+        FP per issue #4239 - jackson-jaxrs (jackson-jaxrs-json-provider) is not jackson-databind
+   ]]></notes>
+        <packageUrl regex="true">^pkg:maven/org\.codehaus\.jackson/jackson-jaxrs@.*$</packageUrl>
+        <cpe>cpe:/a:fasterxml:jackson-databind</cpe>
+    </suppress>
 </suppressions>

--- a/core/src/main/resources/dependencycheck-base-suppression.xml
+++ b/core/src/main/resources/dependencycheck-base-suppression.xml
@@ -4989,4 +4989,11 @@
         <packageUrl regex="true">^pkg:maven/org\.codehaus\.jackson/jackson-jaxrs@.*$</packageUrl>
         <cpe>cpe:/a:fasterxml:jackson-databind</cpe>
     </suppress>
+    <suppress base="true">
+        <notes><![CDATA[
+   FP per issue #4228
+   ]]></notes>
+        <packageUrl regex="true">^pkg:maven/io\.minio/minio@.*$</packageUrl>
+        <cpe>cpe:/a:minio:minio</cpe>
+    </suppress>
 </suppressions>

--- a/core/src/main/resources/dependencycheck-base-suppression.xml
+++ b/core/src/main/resources/dependencycheck-base-suppression.xml
@@ -4996,4 +4996,11 @@
         <packageUrl regex="true">^pkg:maven/com\.hazelcast/hazelcast@(?!5\.1-BETA-1).*$</packageUrl>
         <cve>CVE-2022-0265</cve>
     </suppress>
+    <suppress base="true">
+        <notes><![CDATA[
+        FP per issue #4327
+        ]]></notes>
+        <packageUrl regex="true">^pkg:maven/javax\.ws\.rs/javax\.ws\.rs-api@.*$</packageUrl>
+        <cpe>cpe:/a:eclipse:eclipse_ide</cpe>
+    </suppress>
 </suppressions>

--- a/core/src/main/resources/dependencycheck-base-suppression.xml
+++ b/core/src/main/resources/dependencycheck-base-suppression.xml
@@ -2,6 +2,13 @@
 <suppressions xmlns="https://jeremylong.github.io/DependencyCheck/dependency-suppression.1.3.xsd">
     <suppress base="true">
         <notes><![CDATA[
+            FP per issue #4199
+            ]]></notes>
+        <packageUrl regex="true">^pkg:maven/org\.sonarsource\.scanner\.gradle/sonarqube-gradle-plugin@.*$</packageUrl>
+        <cpe>cpe:/a:sonarsource:sonarqube</cpe>
+    </suppress>
+    <suppress base="true">
+        <notes><![CDATA[
             fp per #3938
             ]]></notes>
         <packageUrl regex="true">^pkg:maven/io\.quarkus/quarkus\-fs\-util@.*$</packageUrl>

--- a/core/src/main/resources/dependencycheck-base-suppression.xml
+++ b/core/src/main/resources/dependencycheck-base-suppression.xml
@@ -648,7 +648,8 @@
             52. tag project is a GO implemention #3047 & #3043
             53. kubernetes:kubernetes is a GO implementation #1035 & #1056 & #1880 & #3529 & #3530 &
             54. aaugustin websockets is a python library #3460
-            54. cron_project:cron is a linux C application #3548
+            55. cron_project:cron is a linux C application #3548
+            56. html2pdf_project:html2pdf is a PHP framework #4021, #4251
         ]]></notes>
         <filePath regex="true">.*(\.(dll|jar|ear|war|pom|nupkg|nuspec|aar)|pom\.xml|package.json|packages.config)$</filePath>
         <cpe>cpe:/a:sandbox:sandbox</cpe>
@@ -706,6 +707,7 @@
         <cpe>cpe:/a:kubernetes:kubernetes</cpe>
         <cpe>cpe:/a:websockets_project:websockets</cpe>
         <cpe>cpe:/a:cron_project:cron</cpe>
+        <cpe>cpe:/a:html2pdf_project:html2pdf</cpe>
     </suppress>
     <suppress base="true">
         <notes><![CDATA[

--- a/core/src/main/resources/dependencycheck-base-suppression.xml
+++ b/core/src/main/resources/dependencycheck-base-suppression.xml
@@ -4996,4 +4996,11 @@
         <packageUrl regex="true">^pkg:maven/io\.minio/minio@.*$</packageUrl>
         <cpe>cpe:/a:minio:minio</cpe>
     </suppress>
+    <suppress base="true">
+        <notes><![CDATA[
+   FP per issue #4227 simple-xml-safe is a security-patched fork of the unmaintained simple-xml
+   ]]></notes>
+        <packageUrl regex="true">^pkg:maven/com\.carrotsearch\.thirdparty/simple-xml-safe@.*$</packageUrl>
+        <cpe>cpe:/a:simplexml_project:simplexml</cpe>
+    </suppress>
 </suppressions>

--- a/core/src/main/resources/dependencycheck-base-suppression.xml
+++ b/core/src/main/resources/dependencycheck-base-suppression.xml
@@ -650,6 +650,7 @@
             54. aaugustin websockets is a python library #3460
             55. cron_project:cron is a linux C application #3548
             56. html2pdf_project:html2pdf is a PHP framework #4021, #4251
+            57. shadow_project:shadow is a suite of C-applications on Linux #4237
         ]]></notes>
         <filePath regex="true">.*(\.(dll|jar|ear|war|pom|nupkg|nuspec|aar)|pom\.xml|package.json|packages.config)$</filePath>
         <cpe>cpe:/a:sandbox:sandbox</cpe>
@@ -708,6 +709,7 @@
         <cpe>cpe:/a:websockets_project:websockets</cpe>
         <cpe>cpe:/a:cron_project:cron</cpe>
         <cpe>cpe:/a:html2pdf_project:html2pdf</cpe>
+        <cpe>cpe:/a:shadow_project:shadow</cpe>
     </suppress>
     <suppress base="true">
         <notes><![CDATA[

--- a/core/src/main/resources/dependencycheck-base-suppression.xml
+++ b/core/src/main/resources/dependencycheck-base-suppression.xml
@@ -4996,4 +4996,11 @@
         <packageUrl regex="true">^pkg:maven/com\.hazelcast/hazelcast@(?!5\.1-BETA-1).*$</packageUrl>
         <cve>CVE-2022-0265</cve>
     </suppress>
+    <suppress base="true">
+        <notes><![CDATA[
+        FP per issue #4327 and #4328
+   ]]></notes>
+        <packageUrl regex="true">^pkg:maven/javax\.ws\.rs/javax\.ws\.rs-api@.*$</packageUrl>
+        <cpe>cpe:/a:eclipse:eclipse_ide</cpe>
+    </suppress>
 </suppressions>

--- a/core/src/main/resources/dependencycheck-base-suppression.xml
+++ b/core/src/main/resources/dependencycheck-base-suppression.xml
@@ -2,6 +2,13 @@
 <suppressions xmlns="https://jeremylong.github.io/DependencyCheck/dependency-suppression.1.3.xsd">
     <suppress base="true">
         <notes><![CDATA[
+            FP per issue #4180
+            ]]></notes>
+        <packageUrl regex="true">^pkg:maven/com\.vaadin/vaadin-spring@.*$</packageUrl>
+        <cpe>cpe:/a:vaadin:vaadin</cpe>
+    </suppress>
+    <suppress base="true">
+        <notes><![CDATA[
             fp per #3938
             ]]></notes>
         <packageUrl regex="true">^pkg:maven/io\.quarkus/quarkus\-fs\-util@.*$</packageUrl>

--- a/core/src/main/resources/dependencycheck-base-suppression.xml
+++ b/core/src/main/resources/dependencycheck-base-suppression.xml
@@ -2936,9 +2936,9 @@
     </suppress>
     <suppress base="true">
         <notes><![CDATA[
-        False positive per issue #1068
+        False positive per issue #1068, #4279
         ]]></notes>
-        <gav regex="true">^(?!(io\.netty|org\.jboss\.netty)).*:.*netty.*$</gav>
+        <packageUrl regex="true">^pkg:maven/(?!(io\.netty|org\.jboss\.netty)).*/.*netty.*$</packageUrl>
         <cpe regex="true">^cpe:/a:netty(_project)?:netty.*$</cpe>
     </suppress>
     <suppress base="true">

--- a/core/src/main/resources/dependencycheck-base-suppression.xml
+++ b/core/src/main/resources/dependencycheck-base-suppression.xml
@@ -5024,4 +5024,11 @@
         <packageUrl regex="true">^pkg:maven/ch\.qos\.reload4j/reload4j@.*$</packageUrl>
         <cpe>cpe:/a:apache:log4j</cpe>
     </suppress>
+    <suppress base="true">
+        <notes><![CDATA[
+   FP per issue #4317
+   ]]></notes>
+        <packageUrl regex="true">^pkg:maven/com\.mattbertolini/liquibase-slf4j@.*$</packageUrl>
+        <cpe>cpe:/a:liquibase:liquibase</cpe>
+    </suppress>
 </suppressions>

--- a/core/src/main/resources/dependencycheck-base-suppression.xml
+++ b/core/src/main/resources/dependencycheck-base-suppression.xml
@@ -4969,4 +4969,11 @@
         <cpe>cpe:/a:davenport:davenport</cpe>
         <cpe>cpe:/a:protonmail:protonmail</cpe>
     </suppress>
+    <suppress base="true">
+        <notes><![CDATA[
+          FP CPE match per #4289
+   ]]></notes>
+        <packageUrl regex="true">^pkg:maven/org\.bouncycastle/bc\-fips@.*$</packageUrl>
+        <cpe>cpe:/a:bouncycastle:legion-of-the-bouncy-castle:</cpe>
+    </suppress>
 </suppressions>

--- a/core/src/main/resources/dependencycheck-base-suppression.xml
+++ b/core/src/main/resources/dependencycheck-base-suppression.xml
@@ -571,6 +571,17 @@
     </suppress>
     <suppress base="true">
         <notes><![CDATA[
+       Spring-framework libraries always have groupId org.springframework with no further extension, so suppress the
+       spring_framework CPE on other Spring libraries
+       Fixes FPs of #4364, #4363, #4362, 4353, #4351, #4350, #4349
+       ]]></notes>
+        <packageUrl regex="true">^pkg:maven/org\.springframework\..*$</packageUrl>
+        <cpe>cpe:/a:springsource:spring_framework</cpe>
+        <cpe>cpe:/a:pivotal_software:spring_framework</cpe>
+        <cpe>cpe:/a:vmware:spring_framework</cpe>
+    </suppress>
+    <suppress base="true">
+        <notes><![CDATA[
        False positive per #1513. Spring-boot-starter-data-rest is not data-rest (however, it does
            depend on spring-data-rest so the actual library will get flagged instead of the "boot" version
            being flagged as spring-data-rest with the wrong version number)

--- a/core/src/main/resources/dependencycheck-base-suppression.xml
+++ b/core/src/main/resources/dependencycheck-base-suppression.xml
@@ -4884,9 +4884,9 @@
     </suppress>
     <suppress base="true">
        <notes><![CDATA[
-       FP per issue #4140
+       FP per issue #4140, 4256
        ]]></notes>
-       <packageUrl regex="true">^pkg:maven/org\.aspectj/aspectjrt@.*$</packageUrl>
+       <packageUrl regex="true">^pkg:maven/org\.aspectj/aspectj.*@.*$</packageUrl>
        <cpe>cpe:/a:vmware:tools</cpe>
     </suppress>
     <suppress base="true">

--- a/core/src/main/resources/dependencycheck-base-suppression.xml
+++ b/core/src/main/resources/dependencycheck-base-suppression.xml
@@ -5044,4 +5044,11 @@
         <packageUrl regex="true">^pkg:maven/com\.mattbertolini/liquibase-slf4j@.*$</packageUrl>
         <cpe>cpe:/a:liquibase:liquibase</cpe>
     </suppress>
+    <suppress base="true">
+        <notes><![CDATA[
+   FP per issue #4344
+   ]]></notes>
+        <packageUrl regex="true">^pkg:maven/com\.pinterest\.ktlint/ktlint-reporter-checkstyle@.*$</packageUrl>
+        <cpe>cpe:/a:checkstyle:checkstyle</cpe>
+    </suppress>
 </suppressions>

--- a/core/src/main/resources/dependencycheck-base-suppression.xml
+++ b/core/src/main/resources/dependencycheck-base-suppression.xml
@@ -1576,6 +1576,13 @@
     </suppress>
     <suppress base="true">
         <notes><![CDATA[
+   FP per issue #4212
+   ]]></notes>
+        <packageUrl regex="true">^pkg:maven/org\.jetbrains\.kotlinx/kotlinx-serialization-core-jvm@.*$</packageUrl>
+        <cpe>cpe:/a:jetbrains:kotlin</cpe>
+    </suppress>
+    <suppress base="true">
+        <notes><![CDATA[
         False positives found while investigating https://github.com/jeremylong/dependency-check-gradle/issues/103
         ]]></notes>
         <gav regex="true">^org\.jetbrains:annotations:.*$</gav>

--- a/core/src/main/resources/dependencycheck-base-suppression.xml
+++ b/core/src/main/resources/dependencycheck-base-suppression.xml
@@ -4961,4 +4961,12 @@
         <packageUrl regex="true">^pkg:maven/org\.apache\.james/apache\-mime4j\-.*$</packageUrl>
         <cpe>cpe:/a:apache:james:</cpe>
     </suppress>
+    <suppress base="true">
+        <notes><![CDATA[
+          log4cats-core_2.13-2.2.0.jar FPs #4295
+   ]]></notes>
+        <packageUrl regex="true">^pkg:maven/org\.typelevel/log4cats\-core_2\.13@.*$</packageUrl>
+        <cpe>cpe:/a:davenport:davenport</cpe>
+        <cpe>cpe:/a:protonmail:protonmail</cpe>
+    </suppress>
 </suppressions>

--- a/core/src/main/resources/dependencycheck-base-suppression.xml
+++ b/core/src/main/resources/dependencycheck-base-suppression.xml
@@ -729,6 +729,7 @@
             17. ws_project is a node websocket client (#1535)
             18. i18n_project is a node.js i18n module (#3350) (#3352) (#3353)
             19. redis:redis is a C project #3744, #3814, #3840, #3841
+            20. perl:perl is a C project #4254
         ]]></notes>
         <filePath regex="true">.*(\.(jar|ear|war|pom)|pom\.xml)$</filePath>
         <cpe>cpe:/a:dash:dash</cpe>
@@ -750,6 +751,7 @@
         <cpe>cpe:/a:ws_project:ws</cpe>
         <cpe>cpe:/a:i18n_project:i18n</cpe>
         <cpe>cpe:/a:redis:redis</cpe>
+        <cpe>cpe:/a:perl:perl</cpe>
     </suppress>
     <suppress base="true">
         <notes><![CDATA[

--- a/core/src/main/resources/dependencycheck-base-suppression.xml
+++ b/core/src/main/resources/dependencycheck-base-suppression.xml
@@ -1562,30 +1562,9 @@
     </suppress>
     <suppress base="true">
         <notes><![CDATA[
-        FP per #2499 and #2600
+        Joint FP CPE suppression for #2346, #2499, #2600, #3274, #4212 also covering future Kotlin extension libraries
         ]]></notes>
-        <packageUrl regex="true">^pkg:maven/org\.jetbrains\.kotlinx/kotlinx\-coroutines\-.*$</packageUrl>
-        <cpe>cpe:/a:jetbrains:kotlin</cpe>
-    </suppress>
-    <suppress base="true">
-        <notes><![CDATA[
-        kotlinx-html-jvm is a kotlin DSL for generating HTML and should not be identified as core kotlin (#2346)
-        ]]></notes>
-        <packageUrl regex="true">^pkg:maven/org\.jetbrains\.kotlinx/kotlinx\-html\-jvm@.*$</packageUrl>
-        <cpe>cpe:/a:jetbrains:kotlin</cpe>
-    </suppress>
-    <suppress base="true">
-        <notes><![CDATA[
-   FP per issue #4212
-   ]]></notes>
-        <packageUrl regex="true">^pkg:maven/org\.jetbrains\.kotlinx/kotlinx-serialization-core-jvm@.*$</packageUrl>
-        <cpe>cpe:/a:jetbrains:kotlin</cpe>
-    </suppress>
-    <suppress base="true">
-        <notes><![CDATA[
-        false positive per #3274 on org.jetbrains.kotlinx:atomicfu
-        ]]></notes>
-        <packageUrl regex="true">^pkg:maven/org\.jetbrains\.kotlinx/atomicfu@.*$</packageUrl>
+        <packageUrl regex="true">^pkg:maven/org\.jetbrains\.kotlinx/.*$</packageUrl>
         <cpe>cpe:/a:jetbrains:kotlin</cpe>
     </suppress>
     <suppress base="true">

